.github/workflows/build.yaml

```diff
@@ -132,3 +134,11 @@ jobs:
         with:
           api-key: ${{ secrets.FOSSA_API_KEY }}
           container: us-docker.pkg.dev/castai-hub/library/agent-rh-ubi:${{ env.RELEASE_TAG }}
+
+      - name: Release
+        uses: softprops/action-gh-release@v2
+        if: github.event_name == 'release'
+        with:
+          files: |
+            bin/castai-agent-amd64
+            bin/castai-agent-arm64
```

Comment:
Can we extract this into `release` workflow instead of adding as a step to build?